Install fails on Windows CI

While it works locally, it fails to install on our CI Jenkins (Windows):

package.json:
`  "dependencies": {
    "odiff-bin": "^2.1.1"
  }`

log:
> odiff-bin@2.1.1 postinstall C:\Jenkins\workspace\Web\CubeMobileApps@3\Webpages\CubeMobileApps\node_modules\odiff-bin
> node ./postinstall.js
> Couldn't find C:\Jenkins\workspace\Web\CubeMobileApps@3\Webpages\CubeMobileApps\node_modules\odiff-bin\platform-windows-x64\bin\odiff trying with .exe
> **error: Esy prefix path is too deep in the filesystem, Esy won't be able to relocate artefacts**
> npm ERR! code ELIFECYCLE
> npm ERR! errno 1
> npm ERR! odiff-bin@2.1.1 postinstall: `node ./postinstall.js`
> npm ERR! Exit status 1
> npm ERR! 
> npm ERR! Failed at the odiff-bin@2.1.1 postinstall script.
> npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

As this is a shared CI setup with other projects, we can't change the directory containing the pipeline workspaces.

--- a/package.json
+++ b/package.json
@@ -7,7 +7,6 @@
     "build": "dune build -p #{self.name}",
     "buildDev": "refmterr dune build --promote-install-files --root . --only-package #{self.name}",
     "release": {
-      "rewritePrefix": true,
       "bin": {
         "odiff": "ODiffBin"
       },

